tests/lit-tests/ssp.ispc

```diff
@@ -0,0 +1,16 @@
+// Make sure the appropriate ssp function attribute is set.
```

Comment:
It would be good to also add a test to check the presence of stack smash protection in the generated asm.

Reply:
+1 here. LLVM may silently break something and do not generate stack protection in the binary. We need to verify that it happens.

Reply:
I am fine with the added tests. Ideally, it would be good to add a test for macOS as well or enable the Linux one (it might be identical).